crates/viewer/re_redap_browser/src/servers.rs

```diff
@@ -69,11 +72,16 @@ impl Server {
     }
 }
 
+pub enum Selection {
+    Dataset(EntryId),
+    Server(re_uri::Origin),
+}
+
```

Comment:
For consistency with naming in the rest of the viewer, I'd call this structure `Item` (similar to `re_viewer_context::Item`). Also, we can use `Entry` here, as tables are coming soon-ish.

```suggestion
pub enum Item {
    Entry(EntryId),
    Server(re_uri::Origin),
}

```